neotron-bmc-pico/build.rs

```diff
@@ -0,0 +1,27 @@
+/// This is the build-script for the Neotron BMC.
```

Comment:
Files should start with `//!` comments. I might have gotten this wrong in the previous version, but we should fix it otherwise it confuses the docs and the code formatter.

`//!` is 'document the thing I am in'
`///` is 'document the thing that comes after'